### PR TITLE
[IMP]10693 add module web_login

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by Rona111** [comment]
cc @elicoidal @duanyp1991 @siyuancn @seb-elico

File: web_login/README.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+Website Login
+=============
+
+This module extends the functionality of website to redirect to the main of the website when log in.
```
Comment (elicoidal) [inline web_login/README.rst]: to the shop of the website

File: web_login/README.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+Website Login
```
Comment (elicoidal) [inline web_login/README.rst]: Redirect to shop after login

**Comment by elicoidal** [comment]
module name: website_redirect_to_shop

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

**Comment by elicoidal** [comment]
LGTM. To be tested in v9 (probably works as well)

File: website_redirect_to_shop/README.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline website_redirect_to_shop/README.rst]: Adapt

File: website_redirect_to_shop/README.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20web_login%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (elicoidal) [inline website_redirect_to_shop/README.rst]: remove this line

File: website_redirect_to_shop/README.rst [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+=============
+
+To configure this module, you need to:
+
+ * No specific configuration needed.
+
+Usage
+=====
+
+For further information, please visit:
```
Comment (elicoidal) [inline website_redirect_to_shop/README.rst]: remove this text and add Runbot

File: website_redirect_to_shop/README.rst [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Redirect to shop after login
+============================
+
+This module extends the functionality of website to redirect to the shop of the website.
+
+Installation
```
Comment (elicoidal) [inline website_redirect_to_shop/README.rst]: remove following 2 sections

File: website_redirect_to_shop/README.rst [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+* Gu SiYuan <gu.siyuan@elico-corp.com>
+
+Maintainer
+----------
+
+.. image:: https://www.elico-corp.com/logo.png
+:alt: Elico Corp
+:target: https://www.elico-corp.com
+
+This module is maintained by Elico Corporation.
```
Comment (elicoidal) [inline website_redirect_to_shop/README.rst:36]: update to newest template

**Review by elicoidal — CHANGES_REQUESTED** [review]
small changes in README

File: website_redirect_to_shop/README.rst (hunk 7 above)
Comment (noahzaozao) [reply website_redirect_to_shop/README.rst:36]: done

**Comment by noahzaozao** [comment]
@elicoidal please review again

**Review by elicoidal — APPROVED** [review]

**Pull request merged**